server/src/instant/db/instaql.clj

```diff
@@ -1173,33 +1173,6 @@
      :forms forms
      :referenced-etypes referenced-etypes}))
 
-(defn clean-where-for-hash [where]
```

Comment:
This just moved into the `instaql-util` file so that I can import it from the hint plan testing ns.